e2e/pages/preferences.ts

```diff
@@ -0,0 +1,44 @@
+import { Page, Locator } from 'playwright';
+export class PreferencesPage {
```

Comment:
This seems to have a lot of things (and most elements can't be on screen at the same time); would it make sense to have sub-pages?